storage/clickhousespanstore/writer.go

```diff
@@ -273,6 +276,12 @@ func uniqueTagsForSpan(span *model.Span) []string {
 	return tags
 }
 
-func tagString(kv *model.KeyValue) string {
-	return fmt.Sprintf("%s=%s", kv.Key, kv.AsString())
+func tagString(buf *strings.Builder, kv *model.KeyValue) string {
```

Comment:
Can you explain why do we need to revert this? 

Also is this method more performant than the one using `fmt`?

Reply:
Yes, it's 2 times faster, that's why I'm revertng this. See comments in #50 for details.

Reply:
thanks for the pointer. 

Could you please run a similar benchmark with using `+` to create the final string? It might be even faster https://dev.to/pmalhaire/concatenate-strings-in-golang-a-quick-benchmark-4ahh.